modules/core/arrow-core/src/test/kotlin/arrow/core/IdTest.kt

```diff
@@ -25,7 +28,6 @@ class IdTest : UnitSpec() {
       SemigroupLaws.laws(Id.semigroup(Int.semigroup()), Id(1), Id(2), Id(3), Id.eq(Int.eq())),
       MonoidLaws.laws(Id.monoid(Int.monoid()), Id(1), Id.eq(Int.eq())),
       ShowLaws.laws(Id.show(), Eq.any()) { Id(it) },
-      MonoidLaws.laws(Id.monoid(String.monoid()), Id("Hello"), Eq.any()),
```

Comment:
???

Reply:
This was overlap between my branch & master. Fixed within merge.